packages/airnode-deployer/terraform/aws/main.tf

```diff
@@ -25,9 +28,12 @@ module "run" {
   }
 }
 
+# We need to list all the possible gateway combinations due to some obscure Terraform bug.
```

Comment:
In Terraform not really. Or at least not that easily. I don't think it would be worth it. Especially since this should be just a temporary state due to a bug.